doc/user/content/sql/create-sink.md

```diff
@@ -159,10 +163,12 @@ Note that:
 
 ### Kafka sinks
 
-When creating Kafka sinks, Materialize uses the Kafka Admin API to create a new topic, and registers its Avro schema in the Confluent Schema Registry. Materialize names the new topic using the format below.
+When creating Avro-formatted Kafka sinks, Materialize uses the Kafka Admin API to create a new topic, and registers its Avro schema in the Confluent Schema Registry. Materialize names the new topic using the format below.
```

Comment:
I just realized that this is outdated:

> Materialize uses the Kafka Admin API to create a new topic

With `reuse_topic=true` we only create a new topic if it doesn't exist already.

Also, we could split this part up into the topic part and the schema part. For both Avro and JSON we create the topics, but only for Avro do we register the schema.

Another side note: we could also support publishing a JSON schema to the schema registry, I think it should support it.

Reply:
I've updated this bit! I think I've covered everything.

> Another side note: we could also support publishing a JSON schema to the schema registry, I think it should support it.

Also, agreed! I'm actually working on Protobuf + schema registry integration next, and the goal is to make that area of the code more generic so that JSON will be a quick add on. Hopefully I will be updating this part of the docs again shortly 😄